### PR TITLE
[IMP] purchase_last_price_info: Add last_purchase_price field dependa…

### PR DESCRIPTION
**Pull request by Daniel-CA**
…t to the "Product Price" decimal precision

### CODE REVIEW

**Review by oihane — APPROVED** [review]
LGTM

File: purchase_last_price_info/__manifest__.py [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 {
     "name": "Purchase Product Last Price Info",
-    "version": "12.0.1.0.0",
+    "version": "12.0.1.1.0",
```
Comment (bodedra) [inline purchase_last_price_info/__manifest__.py]: Remove this changes. It will taken care by merge command.

**Review by bodedra — APPROVED** [review]
LG :+1:

File: purchase_last_price_info/__manifest__.py (hunk 1 above)
Comment (Daniel-CA) [reply purchase_last_price_info/__manifest__.py]: > Remove this changes. It will taken care by merge command.

Done!

**Review by bodedra — APPROVED** [review]

**Review by HviorForgeFlow — APPROVED** [review]

**Comment by HviorForgeFlow** [comment]
Have you tried if this change have side effects when updating the module from previous version?

**Comment by Daniel-CA** [comment]
Hi @HviorForgeFlow, yes, there is no side effects, but you should update the field column to view the changes if you have a different decimal precision than the default.

**Comment by HviorForgeFlow** [comment]
Manually? What happens if you have the module installed in master version and you apply this update. 

Do you lost column data, or nothing happens until you recompute the price?

 If it is the second option maybe we need to apply some migration script through this version.

**Review by HviorForgeFlow — CHANGES_REQUESTED** [review]
Waiting an answer

**Review by bodedra — CHANGES_REQUESTED** [review]
Agree with @HviorForgeFlow comment.

**Comment by HviorForgeFlow** [comment]
ping @Daniel-CA

**Comment by Daniel-CA** [comment]
Hey there, I have this issue pending, there is no data lost but I was going to make a script to update the column data format to match like it should be.

**Comment by Daniel-CA** [comment]
I have been testing previous version and looks like that there is no need of change the column data format, it works like it is, looks like that in my previous test I didn't reload the view. There is no new data recalculation or anything similar, just updates the view to match with the current "product price" decimal precision.

**Review by HviorForgeFlow — APPROVED** [review]
Ok, fair enough :smile:

**Review by bodedra — APPROVED** [review]

**Review by marcelsavegnago — APPROVED** [review]

**Pull request merged**